common/font.go

```diff
@@ -248,6 +248,11 @@ func (f *Font) generateFontAtlas(c int) FontAtlas {
 	return atlas
 }
 
+// Should only use this if you are using your own custom text shader.
+func (f *Font) GenerateFontAtlas(c int) FontAtlas {
```

Comment:
For brevity I would just capitalize the original function rather than create a separate exported function. That said, the go standard library actually does this often so this should be fine.